src/pages/using-spark/components/accordion.mdx

```diff
@@ -46,14 +35,13 @@ but the entire header area is clickable for the same action.
 
 ## Anatomy
 
+- An Accordion Item must have a clickable header that reveals or hides information.
 - An Accordion Item must have a left aligned title.
 - An Accordion Item must contain content.
-- An Accordion Item must have a right aligned
-Chevron-[Up/Down]-Circle-Two-Color Icon
+- An Accordion Item must have a right aligned Chevron-[Up/Down]-Circle-Two-Color Icon
```

Comment:
We removed all the two color icons. This needs updating

Reply:
Taking care of this! Will check for two-color throughout the code base as well